FlowCrypt/Functionality/Services/Local Private Key Services/CombinedPassPhraseStorage.swift

```diff
@@ -35,12 +35,9 @@ struct PassPhrase: Codable, Hashable, Equatable {
         PassPhrase(value: self.value, email: self.email, fingerprintsOfAssociatedKey: self.fingerprintsOfAssociatedKey, date: Date())
     }
 
-    // We still need == operator here because we use `withUpdatedDate` to set `date` field to up-to-date
-    // Therfore, 2 passphrases might be treated differently even though
-    // they are exactly same if we don't implement custom == operator
+    // Two pass phrases are treated same when their email and fingerprints are same
     static func == (lhs: PassPhrase, rhs: PassPhrase) -> Bool {
         return lhs.primaryFingerprintOfAssociatedKey == rhs.primaryFingerprintOfAssociatedKey
-                && lhs.value == rhs.value
```

Comment:
when updating passphrase if we want to compare `value` too, old passphrase will be treated different than new passphrase even though they are actually same fingerprint & emails.

Reply:
But I think here we'll be updating only `date`, as passphrase right `value` will be the same.

Reply:
Let's say old saved passphrase value is `test` and new passphrase value is `test`(those fingerprints are same)
When we want to update this passphrase with `passPhrases.update(with: passPhrase)`, it will treat like those 2 passphrases are different and therefore 2 passphrases will be created.
I don't think we need 2 passphrases for same fingerprint.

Reply:
Of course, old passphrase value and new passphrase value field will be same. But just in case

Reply:
I'll revert changes if you want so

Reply:
Yeah, let's leave as it was before

Reply:
ok